modules/cudf/test/cudf-data-frame-tests.ts

```diff
@@ -426,6 +427,20 @@ test(
     expect(result.names).toEqual([]);
   });
 
+describe('dataframe mathematicalUnaryOps', () => {
+  const a  = Series.new({type: new Int32, data: [-3, 0, 3]});
+  const b  = Series.new({type: new Int8, data: [-3, 0, 3]});
+  const c  = Series.new(['foo', null, 'bar']);
+  const df = new DataFrame({'a': a, 'b': b, 'c': c});
+
+  test('dataframe.acos', () => {
+    const result = df.acos();
+    expect([...result.get('a')]).toEqual([-2147483648, 1, -2147483648]);
+    expect([...result.get('b')]).toEqual([0, 1, 0]);
+    expect([...result.get('c')]).toEqual([...c]);
```

Comment:
Could add a floating point series, confirming that the value in the returned result dataframe is the same as calling `acos` on the series directly:
```
const d  = Series.new({type: new Float32, data: [-2.7, 0, 3.1]});

expect([...result.get('d')]).toEqual([...d.acos()]);
```
This is not as nice as a direct test of values but it does least ensure that the DataFrame and Series calculations on floating point are consistent with each other